dashboard/test/ui/features/projects.feature

```diff
@@ -37,8 +40,26 @@ Scenario: Applab Flow
   And I click selector ".project_save"
   And I wait until element ".project_edit" is visible
   Then I should see title "Code Ninja - App Lab"
-  Then I click selector ".project_share"
-  And I wait to see "#x-close"
+
+  # publish and unpublish the project from the share dialog
+
+  Then I open the project share dialog
+  And the project is unpublished
+  And I publish the project from the share dialog
+  And I open the project share dialog
+  And the project is published
```

Comment:
Likewise I feel like each of these could read as its own tiny user story (though still functionally equivalent):

```gherkin
Given I open the project share dialog
And the project is unpublished
When I publish the project from the share dialog
Then I open the project share dialog
And the project is published
```